lib/offsite_payments/integrations/mollie.rb

```diff
@@ -0,0 +1,58 @@
+module OffsitePayments #:nodoc:
+  module Integrations #:nodoc:
+    module Mollie
+      class API
+        include ActiveUtils::PostsData
+
+        attr_reader :token
+
+        def initialize(token)
+          @token = token
+        end
+
+        def get_request(resource, params = nil)
+          uri = URI.parse(MOLLIE_API_V1_URI + resource)
+          uri.query = params.map { |k,v| "#{CGI.escape(k)}=#{CGI.escape(v)}}"}.join('&') if params
+          headers = { "Authorization" => "Bearer #{token}", "Content-Type" => "application/json" }
```

Comment:
This is the same for `get_request` and `post_request`. Maybe makes sense to pull out into a method?